src/Agent/NewRelic/Agent/Core/Agent.cs

```diff
@@ -420,6 +420,15 @@ public void RecordLogMessage(string frameworkName, object logEvent, Func<object,
                 normalizedLevel = string.IsNullOrWhiteSpace(level) ? "UNKNOWN" : level.ToUpper();
             }
 
+            // LogLevelDenyList is already uppercase
```

Comment:
Not sure I understand your comment here. What would you do differently?

Reply:
If there's nothing in the deny list, we don't need to check if `normalizedLevel` is blank, or do the `Contains()` call. I'm sure it won't make any practical difference here, but I think it's a good thing to keep in mind.

Reply:
Gotcha. 

I would expect `HashSet.Contains()` to be pretty smart when there's no data, so that checking `HashSet.Count > 0` might not be any more efficient. 

I also pushed a change to move where we are checking the deny list so that the `string.Empty` isn't necessary at all.